source/documentation/reference/operational-processes.md

```diff
@@ -0,0 +1,174 @@
+# Cloud Platform Operational Processes
+
+This is a record of the operational processes that we will use to support our users and to maintain the cloud platform.
+
+## Hours of support
+
+Our hours of providing support are 10am - 5pm. During this time we will work on support requests from teams and make sure someone is available to answer questions in [`#ask-cloud-platform`](https://mojdt.slack.com/messages/C57UPMZLY/) and look at PR request notifications for the cloud-platform-environments repo on [`#cloud-platform-notify`](https://mojdt.slack.com/messages/CA5MDLM34/)
+
+Outside of these hours we will respond to [high priority](#prioritising-incidents) incidents as per our [on call](#our-on-call-process) process.
+
+## Working hours
+
+A member of the support team will be available online from 9am and until 6pm. This team member can be working remotely. We are doing this so that we cover incident response for the time when the person on call is coming into the office or going home at the end of the day.
+
+This support team member should be ready to respond to any high priority incidents.
+
+## The support sub team
+
+The support sub team will consist of 1-2 engineers. Each cloud platform team member will be on the sub team on a regular basis.
+
+The key activities for the day are:
+
+_On starting the day (9AM)_
+At least one engineer to:
+* Check the [`#low-priority-alarms`](https://mojdt.slack.com/messages/C8QR5FQRX/) and [`#high-priority-alarms`](https://mojdt.slack.com/messages/C8PF51AT0/) slack channels for any issues to investigate
```

Comment:
This bullet list formatting is broken. I think you need a blank line before the list.
<img width="896" alt="Screen Shot 2019-06-26 at 16 48 04" src="https://user-images.githubusercontent.com/2338/60194997-441c4180-9832-11e9-93b9-c93b2ee4e8b8.png">